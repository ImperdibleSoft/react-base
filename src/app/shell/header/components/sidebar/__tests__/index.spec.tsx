import * as React from 'react';
import { shallow } from 'enzyme';

import Sidebar from '..';

describe('Sidebar', () => {
  it('should render a valid closed sidebar', () => {
    const component = shallow(
      <Sidebar handleClick={() => {}} isOpen={false}>
        <ul>
          <li>Test</li>
        </ul>
      </Sidebar>
    );

    expect(component).toMatchSnapshot();
  });

  it('should render a valid open sidebar', () => {
    const component = shallow(
      <Sidebar handleClick={() => {}} isOpen={true}>
        <ul>
          <li>Test</li>
        </ul>
      </Sidebar>
    );

    expect(component).toMatchSnapshot();
  });
});
