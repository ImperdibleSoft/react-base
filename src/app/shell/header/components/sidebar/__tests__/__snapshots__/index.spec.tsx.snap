// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar should render a valid closed sidebar 1`] = `
<Wrapper>
  <Toggler
    onClick={[Function]}
  >
    <i
      className="fa fa-bars"
    />
  </Toggler>
</Wrapper>
`;

exports[`Sidebar should render a valid open sidebar 1`] = `
<Wrapper>
  <Toggler
    onClick={[Function]}
  >
    <i
      className="fa fa-bars"
    />
  </Toggler>
</Wrapper>
`;
